courses/views.py

```diff
@@ -107,6 +112,36 @@ def course_detail(request: HttpRequest, course_id: int) -> HttpResponse:
     return render(request, "courses/course_detail.html", context)
 
 
+def course_ical(request: HttpRequest, course_id: int) -> HttpResponse:
+    course = get_object_or_404(Course.objects, id=course_id)
+    cal = Calendar()
+    cal.add('version', '2.0')
+    cal.add('prodid', "-//Tanzquotient calendar {}//mxm.dk//".format(course_id))
+    lessons = course.get_lessons()
+    teachers = course.get_teachers()
+    for lesson in lessons:
+        event = Event()
+        event['dtstart'] = vDatetime(lesson.time_from)
+        event['dtend'] = vDatetime(lesson.time_to)
+        event.add('summary', vText(course.name))
+        event.add('description', vText(course.format_description()))
+        event.add('location', vText(course.room))
+        # add teachers as attendees..
+        # Could also add the first one as origanizer if needed
+        # Could possibly also add students, though careful with privacy
+        for teacher in teachers:
+            if teacher.email is not None:
+                # TODO: decide whether exposing the E-Mail is ok, or if we should hide 
```

Comment:
Exposing the teacher's email is not okay. It's their private email and not a public tanzquotient email. Students should always contact the tanzquotient administration (board/team) and not individual teachers.